bootsoas.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+  <head>
+	   <meta charset="utf-8">
+	   <meta name="viewport" content="width=device-width, initial-scale=1">
+	   <link rel="stylesheet" href="css/main.css" />
+		 <link id="defaultIcon" rel="shortcut icon" href="/assets/favicon_06.png" />
+     <title>Booting Sugar on a Stick</title>
+  </head>
+  <body class="content-page">
+    <nav>
+  <div>
+    <a class="red" href="https://activities.sugarlabs.org/">Activities</a>
+    <a class="orange" href="http://wiki.sugarlabs.org/">Wiki</a>
+    <a class="purple last" href="https://socialhelp.sugarlabs.org/">Social Help</a>
+  </div>
+</nav>
+<link rel="stylesheet" type="text/css" href="css/main.css">
+
+<div class="grid">
+  <img class="col col-2-3" src="assets/soaspic.jpg">
+  
+  <p class="col col-1-3">
+  <h2>Sugar Stick</h2>
+    To boot sugar labs OS on your computer, you will need a bootable sugar on stick set-up already.<br>
+    To see how to set it up, visit the <a href="http://wiki.sugarlabs.org/go/Sugar_on_a_Stick/Installation" target="_blank">wiki.</a> 
+  </p>
+</div>
+
+<div class="grid">
+<img class="col col-2-3" src="assets/boot1.jpg" style="height:390px">
+  <p class="col col-1-3">
+  <h2>Plug it In</h2>
+  Once you have your sugar stick, plug it in to a USB port on your computer.<br>
+    The picture on the right will show you what it should look like.
+  </p>
+  
+</div>
+
+<div class="grid">
+  <img class="col col-2-3" src="assets/boot2.jpg">
+  <p class="col col-1-3">
+  <h2>Boot Device Select(Windows)</h2>
```

Comment:
Please but a space, and maybe use `<nobold>`  (or whatever it was on the inquiry page) for the (Windows).

Also, maybe say Some Windows 8+ Devices, b/c older windows computers use tranditional boot menu.